pycm/pycm_util.py

```diff
@@ -775,9 +775,37 @@ def inner_function(*args, **kwargs):
         :type args: list
         :param kwargs: keyword arguments
         :type kwargs: dict
-        :return: None
+        :return: modified function result
         """
         if args[0].metrics_off:
             raise pycmMatrixError(METRICS_OFF_ERROR)
         return func(*args, **kwargs)
     return inner_function
+
+
+def deprecated(func):
+    """
+    Send a warning regarding decorated function's deprecation.
+
+    :param func: input function
+    :type func: function
+    :return: inner function
+    """
+    @wraps(func)
+    def inner_function(*args, **kwargs):
+        """
+        Inner function.
```

Comment:
Please add more details here.

Reply:
I think that's enough. `inner_function` in `metrics_off_check` is similar to this function and we've merged its PR.

Reply:
You’re right. Let’s keep it same. We can fix them if it’s necessary in another PR